src/nuc_eos/readtable.F90

```diff
@@ -24,13 +24,13 @@ subroutine readtable(eos_filename)
 
   eos_filename_stored = eos_filename
 
-  !write(*,*) "Reading Ott EOS Table"
+  write(*,*) "Reading Ott EOS Table"
 
   call h5open_f(error)
 
   call h5fopen_f (trim(adjustl(eos_filename)), H5F_ACC_RDONLY_F, file_id, error)
 
-  !write(6,*) trim(adjustl(eos_filename))
+  write(6,*) trim(adjustl(eos_filename))
```

Comment:
this file also was recently changed in master, from a commit by Sherwood.  Can you match that version please.